cfg/validate_test.go

```diff
@@ -224,6 +238,28 @@ func TestValidateConfigUnsuccessful(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "negative kernel list cache ttl",
+			config: &Config{
+				Logging:   LoggingConfig{LogRotate: validLogRotateConfig()},
+				FileCache: validFileCacheConfig(t),
+				MetadataCache: MetadataCacheConfig{
+					ExperimentalMetadataPrefetchOnMount: "sync",
+				},
+				FileSystem: FileSystemConfig{KernelListCacheTtlSecs: -2},
+			},
+		},
+		{
+			name: "large kernel list cache ttl",
+			config: &Config{
+				Logging:   LoggingConfig{LogRotate: validLogRotateConfig()},
+				FileCache: validFileCacheConfig(t),
+				MetadataCache: MetadataCacheConfig{
+					ExperimentalMetadataPrefetchOnMount: "sync",
+				},
+				FileSystem: FileSystemConfig{KernelListCacheTtlSecs: 88888888888888888},
+			},
+		},
 	}
 
 	for _, tc := range testCases {
```

Comment:
You can combine the following lines as:
assert.Error(t, ValidateConfig(tc.config))